src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Event.java

```diff
@@ -10,4 +10,5 @@
     DOC_CONSUMED, // client service handled the documents
     BLOB_DELETE_FAILURE, // when blob is not successfully deleted after processing is complete
```

Comment:
not related to this PR, but is this still needed/used?

Reply:
That's an event not an envelope status. It is still used as we might want to track how many times blob delete fails.

Reply:
ok